bin/emqx

```diff
@@ -263,7 +263,7 @@ generate_config() {
             ## if they are different
             if [ -n "$TMP_ARG_VALUE" ]; then
                 ## if the old value is present, replace it with generated value
-                sh -c "$SED_REPLACE 's/^$ARG_KEY.*$/$ARG_LINE/' $TMP_ARG_FILE"
+                sh -c "$SED_REPLACE 's|^$ARG_KEY.*$|$ARG_LINE|' $TMP_ARG_FILE"
```

Comment:
@zmstone
 I don't understand why I have to use `sh-c` to execute sed, I tried to delete it and sed command will repo the error  `data/configs/vm.args.tmp: No such file or directory`, but this file is real exist